### PR TITLE
Fix builds by installing radare.r2 manually

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](../DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](../DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](../../radare2book) with the relevant information (if needed)

**Detailed description**

vpm seems to be down:

vpm_down

so this pr tries to fix:

v_install_err

by installing radare.r2 manually if so.

**Test plan**

Builds that use V (e.g. Travis) should be green.

**Closing issues**

See description.

### CODE REVIEW

File: test/src/Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ $(V): v
 	cd v && git pull
 	cd v && git reset --hard $(VCOMMIT)
 	$(MAKE) -C v
-	$(V) install radare.r2
+	$(V) install radare.r2 || git clone --depth=1 https://github.com/radare/v-r2 ~/.vmodules/radare/r2
```
Comment (radare) [inline test/src/Makefile]: why v install fails?

Comment (kazarmy) [reply test/src/Makefile]: no idea ... https://vpm.best/jsmod/radare.r2 is giving me:

but the server is actually returning 200.

**Comment by radare** [comment]
seems like vpm is down 
Screenshot 2020-02-20 at 16 23 59

**Comment by radare** [comment]
yes, why not  :) but then the modulename must be radareorg instead

**Pull request merged**